Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -544,7 +544,6 @@ void CFrame::PopulateToolbar(wxToolBar* ToolBar)
 
 	WxUtils::AddToolbarButton(ToolBar, wxID_OPEN,               _("Open"),        m_Bitmaps[Toolbar_FileOpen],    _("Open file..."));
 	WxUtils::AddToolbarButton(ToolBar, wxID_REFRESH,            _("Refresh"),     m_Bitmaps[Toolbar_Refresh],     _("Refresh game list"));
-	WxUtils::AddToolbarButton(ToolBar, IDM_BROWSE,              _("Browse"),      m_Bitmaps[Toolbar_Browse],      _("Browse for an ISO directory..."));
```

Comment:
The bitmap can only be removed if we stop using it in DolphinQt2: https://github.com/dolphin-emu/dolphin/blob/3ff56aa192f952cb6ad2e9c662d4a368efd21e6f/Source/Core/DolphinQt2/ToolBar.cpp#L83